zanata-war/src/main/java/org/zanata/dao/LocaleMemberDAO.java

```diff
@@ -56,6 +56,24 @@ public LocaleMemberDAO(Session session) {
         return query.list();
     }
 
+    public List<HLocaleMember> findActiveMembers(LocaleId localeId) {
+        StringBuilder sb = new StringBuilder();
+        sb.append("from HLocaleMember ")
+            .append("where id.supportedLanguage.localeId = :localeId ")
+            .append("and (isTranslator = :isTranslator ")
+            .append("or isReviewer = :isReviewer ")
+            .append("or isCoordinator = :isCoordinator) ")
```

Comment:
Is this condition needed at all?

```java
.append("and (isTranslator = :isTranslator ")
.append("or isReviewer = :isReviewer ")
.append("or isCoordinator = :isCoordinator) ")
```
Is there a case where we have a `HLocaleMember` with all those flags turned off?